### DISCUSSION TITLE
Small latex tip

### DISCUSSION THREAD

**Discussion by rubinduck** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#15] [General]
I saw you using \nonumber inside latex flalign environment on every line. It can be replaced by writing flalign* instead of flalign. This way you don't have to insert \nonumber on every line which is convenient.
[More info about it](https://tex.stackexchange.com/questions/120013/multiline-equation-without-number)

**Comment by fifthist** [comment]
Thanks for the tip! I'll give it a shot during an upcoming write-up.
